crates/core/src/factories/mem_peer_store/test.rs

```diff
@@ -10,14 +10,22 @@ const AGENT_1: AgentId = AgentId(Id(bytes::Bytes::from_static(b"agent1")));
 const AGENT_2: AgentId = AgentId(Id(bytes::Bytes::from_static(b"agent2")));
 const SPACE_1: SpaceId = SpaceId(Id(bytes::Bytes::from_static(b"space1")));
 
+fn sneak_url(s: &str) -> Url {
```

Comment:
Not following these function names?

Reply:
hehe. the test needs to sneak in some debug data to tally up which infos are still around or whatever, so I'm putting that data into the url and extracting it again. I can add some docs there.

Reply:
Done: https://github.com/holochain/kitsune2/pull/44/commits/81dcec59fe12ab9b915e59646055d70df8abd8f5